client/core/trade.go

```diff
@@ -2578,11 +2578,6 @@ func (t *trackedTrade) confirmRedemption(match *matchTracker) {
 	}
 
 	if redemptionResubmitted {
-		if match.Side == order.Maker {
-			match.MetaData.Proof.MakerRedeem = order.CoinID(redemptionStatus.CoinID)
-		} else {
-			match.MetaData.Proof.TakerRedeem = order.CoinID(redemptionStatus.CoinID)
-		}
```

Comment:
I was depending on this for #1752 was it a problem? Will the old coin ID stay in here now?

Reply:
oh, sorry, this is the duplicate code I suggested to remove, please ignore